sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/DoubleUpDownCounterSdk.java

```diff
@@ -12,28 +12,26 @@
 import io.opentelemetry.api.metrics.LongUpDownCounterBuilder;
 import io.opentelemetry.api.metrics.ObservableDoubleMeasurement;
 import io.opentelemetry.context.Context;
-import io.opentelemetry.sdk.metrics.aggregator.AggregatorHandle;
 import io.opentelemetry.sdk.metrics.common.InstrumentDescriptor;
 import io.opentelemetry.sdk.metrics.common.InstrumentType;
 import io.opentelemetry.sdk.metrics.common.InstrumentValueType;
+import io.opentelemetry.sdk.metrics.internal.state.BoundStorageHandle;
+import io.opentelemetry.sdk.metrics.internal.state.MeterProviderSharedState;
+import io.opentelemetry.sdk.metrics.internal.state.MeterSharedState;
+import io.opentelemetry.sdk.metrics.internal.state.WriteableMetricStorage;
 import java.util.function.Consumer;
 
-final class DoubleUpDownCounterSdk extends AbstractSynchronousInstrument
-    implements DoubleUpDownCounter {
+final class DoubleUpDownCounterSdk extends AbstractInstrument implements DoubleUpDownCounter {
```

Comment:
For a future PR, I'd prefer that we rename all of these classes to have the `Sdk` as a prefix, rather than a suffix. This isn't an "Sdk" of type "DoubleUpDownCounter", it's the Sdk's implementation of the interface, so using "Sdk" as a prefix makes more sense to me and is consistent with how we ended up naming classes in the tracing SDK.

Reply:
I'm totally on-board.  If you look at my wip-metrics branch I had done that, and retrofitting commits against main has been hard due to it.  I also prefer `Sdk*`.